app/v2/controllers/ConvertError.scala

```diff
@@ -51,84 +64,52 @@ trait ConvertError extends Logging {
     def convert(input: E): PresentationError
   }
 
-  val jsonToXmlValidationErrorConverter = new Converter[FailedToValidateError] {
-    import v2.models.errors.FailedToValidateError._
```

Comment:
I think we can keep import at block level if it's not reported in warnings. That's look simple and more understandable.

Reply:
This has already been changed 👍